SingularityService/src/main/java/com/hubspot/singularity/event/SingularityEventController.java

```diff
@@ -144,6 +145,28 @@ public Void call() {
     processFutures(builder.build());
   }
 
+  @Override
+  public void elevatedAccessEvent(ElevatedAccessEvent elevatedAccessEvent) {
+    ImmutableSet.Builder<ListenableFuture<Void>> builder = ImmutableSet.builder();
+
+    for (final SingularityEventSender eventListener : eventListeners) {
+      builder.add(
+        listenerExecutorService.submit(
+          new Callable<Void>() {
```

Comment:
feel free to change any of these to lambdas too. Much of this is pre-java8 code which is why there are callables/runnables all over